recipes/metaclock/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "metaclock" %}
+{% set version = "1.0.0" %}
+
+
+
+package:
+  name:  "{{ name }}"
+  version:  "{{ version }}"
+
+source:
+  url: https://github.com/SegataLab/metaclock/archive/{{ version }}.tar.gz
+  sha256: 146e6e8b5c8d62dc958a41cfb5a53cae9cc57bbc1da82d4020b217aafc180429
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --ignore-installed --no-deps -vv"
+
+requirements:
+  host:
+    - python =3.6.10
+    - setuptools
+    - numpy
+    - cython
+    - pip
+  run:
+    - python =3.6.10
+    - numpy 
+    - matplotlib-base >=3.2.1
+    - seaborn >=0.10.1 
+    - samtools
+    - bzip2 =1.0.8
```

Comment:
```suggestion
    - bzip2
```

Reply:
pin has been removed and other pins too. Thank you for this suggestion so that the hanging problem in 'build' could be solved.